### ISSUE TITLE
forcing DATM with cpl history files requires concatenating the files first

### ISSUE THREAD

**Issue by adamrher** [ESMCI/cime#3458]
The histaux_a2x3hr = .true. namelist provides cpl i/o with date stamps in the form of year-month-day-[seconds-of-first-timeslice], and each file contains 1 day of i/o. But forcing clm with these files in a datm configuration (i.e., running a DATM%CPLHIST CTSM I-compset) assumes the cpl i/o are each monthly files. 

These are the XLM variables I set for a test with 2 years of forcing files, in running the I-compset:

`DATM_CPLHIST_DIR='/glade/scratch/aherring/cam6_2_017_FHIST_f09_f09_mg17_200316_1979-a2x3hr-2yr/run/'`
`DATM_CPLHIST_CASE='cam6_2_017_FHIST_f09_f09_mg17_200316_1979-a2x3hr-2yr'`
`DATM_CPLHIST_YR_ALIGN=1979`
`DATM_CPLHIST_YR_START=1979`
`DATM_CPLHIST_YR_END=1980`

This generates CaseDocs/data.streams.txt* that assume the cpl forcing files use time-stamps as year-month. 

@mvertens commented to me: "@klindsay28 has a utility that concatenates the output histaux_xxx files into files that can be used by datm. So you are right - you need this step before datm can use the files. It would be good to both document this as well as integrate the tool that @klindsay28 has into cime. @klindsay28 - can you please provide details ..."

I agree the tool should be documented and integrated into cime.

FYI. Here is a $CASEROOT where you can see the problem in CaseDocs/data.streams.txt* (i.e., compare the files listed to the files in filepath):

`/glade/scratch/aherring/ctsm1.0.dev079.se_grids.n02_I1850Clm50BgcSpinup_f09_f09_mg17_200317_ioproblem-test`

**Comment by adamrher** [comment]
also FYI. Here is the documentation @ekluzek sent me 2 years ago that I've been using. I don't know if there is more up to date documentation available.

**Comment by adamrher** [comment]
I made a shell script that converts all the output to the correct naming convention: 

`/glade/work/aherring/grids/clm-spinup-check/rename-for-cplhist.sh`

However I ran into a snafu that could potentially be a bug in the i/o files generated through the histaux_xxx namelist vars.

Some context. DATM%CPLHIST mode requires more than just 3-hourly output ....  One needs to invoke these four (4) namelists to provide forcing files for DATM%CPLHIST mode:

`histaux_a2x3hr=.true.`
`histaux_a2x24hr=.true.`
`histaux_a2x1hr=.true.`
`histaux_a2x1hri=.true.`

OK fine. But the problem I've found is that the timestamps of the daily i/o files, invoked by histaux_a2x24hr, are off by 1 day. For a FHIST run the first time-stamp on the daily files is not 1979-01-01 .... but rather 1970-01-02:

<img width="1238" alt="Screen Shot 2020-03-23 at 6 56 46 PM" src="https://user-images.githubusercontent.com/29961476/77377216-166c1680-6d38-11ea-9c26-03b4c7e617dd.png">

but the time index indicates this refers to the previous day:

<img width="1195" alt="Screen Shot 2020-03-23 at 6 59 16 PM" src="https://user-images.githubusercontent.com/29961476/77377351-6d71eb80-6d38-11ea-9bc4-4e37a3e90c31.png">

Likewise, for the last day of the year, 1979-12-31:

<img width="1168" alt="Screen Shot 2020-03-23 at 7 01 05 PM" src="https://user-images.githubusercontent.com/29961476/77377445-b9bd2b80-6d38-11ea-9b43-10c167486af5.png">

Everything is off by one (1) day with the daily files ... unless I'm missing something here?
